products/sle/main.pm

```diff
@@ -678,6 +678,15 @@ my $serial_failures = [];
 if (is_sle('=12-SP4') && check_var('ARCH', 'aarch64')) {
     push @$serial_failures, {type => 'hard', message => 'bsc#1093797', pattern => quotemeta 'Internal error: Oops: 96000006'};
 }
+if(get_var('TRINITY')){
+    push @$serial_failures, {type => 'hard', message => 'Kernel Ooops found', pattern => quotemeta 'Oops:'};
+    push @$serial_failures, {type => 'hard', message => 'Kernel BUG found', pattern => quotemeta 'kernel BUG at'};
+    push @$serial_failures, {type => 'hard', message => 'Kernel BUG found', pattern => quotemeta 'Kernel BUG at'};
```

Comment:
You have some duplicates here. If you put the message for multiple patterns in one line (#L687).